pkg/server/user.go

```diff
@@ -559,6 +559,8 @@ func (s *ImmuServer) addUserToLoginList(u *auth.User) {
 }
 
 func (s *ImmuServer) getLoggedInUserdataFromCtx(ctx context.Context) (int64, *auth.User, error) {
+	s.userdata.Lock()
```

Comment:
sync may be done in `getLoggedInUserDataFromUsername` function

Reply:
done